src/gpl/src/placerBase.cpp

```diff
@@ -1270,27 +1270,29 @@ void PlacerBase::printInfo() const
   dbBlock* block = db_->getChip()->getBlock();
   log_->info(GPL,
              6,
-             "{:20} {:10}",
-             "NumInstances:",
+             "{:27} {:10}",
+             "Total GPL instances:",
```

Comment:
A "GPL instance" is unclear.  I'm not even sure what it is so I don't users will undertand.